modules/oc-mirror-mirror-to-mirror.adoc

```diff
@@ -2,15 +2,14 @@
 //
 // * installing/disconnected_install/installing-mirroring-disconnected.adoc
 // * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
-// * microshift_running_apps/microshift_operators/microshift-operators-oc-mirror.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="oc-mirror-mirror-to-mirror_{context}"]
 = Mirroring from mirror to mirror
 
 You can use the oc-mirror plugin to mirror an image set directly to a target mirror registry that is accessible during image set creation.
 
-You are required to specify a storage backend in the image set configuration file. This storage backend can be a local directory or a Docker v2 registry. The oc-mirror plugin stores metadata in this storage backend during image set creation.
+You must specify a storage backend in the image set configuration file. This storage backend can be a local directory or a Docker v2 registry. The oc-mirror plugin stores metadata in this storage backend during image set creation.
```

Comment:
why storage backend again ?

Reply:
Yeah, this is an OCP module, so I can't answer that. This is why I hate using OCP content.